pkg/defaultPolicies/defaultPolicies.yaml

```diff
@@ -385,4 +385,26 @@ policies:
       - identifier: ALL_EXPOSED_SECRET_VAULT
         messageOnFailure: Secret data found in config - keep your sensitive data elsewhere to prevent it from being stolen
       - identifier: ALL_EXPOSED_SECRET_PRIVATEKEY
-        messageOnFailure: Secret data found in config - keep your sensitive data elsewhere to prevent it from being stolen  
\ No newline at end of file
+        messageOnFailure: Secret data found in config - keep your sensitive data elsewhere to prevent it from being stolen
+  - name: EKS
```

Comment:
you can test it out by running `datree publish ./pkg/defaultPolicies/defaultPolicies.yaml` and then accessing the dashboard or running `datree test`